src/Map.jsx

```diff
@@ -12,6 +12,42 @@ const initialStyle = {
   height: "100%",
 };
 
+const ZoomControl = ({ home }) => {
+  const map = useMap();
+  return (
+    <div className="leaflet-bottom leaflet-right">
```

Comment:
Can these be `Box`?

Reply:
Also, for reference -- are you able to add links to the CSS with the values for these classes?

Reply:
Yes, added a comment that links to the leaflet css file